app/models/validations/sales/household_validations.rb

```diff
@@ -46,6 +47,21 @@ def validate_person_age_and_relationship_matches_economic_status(record, person_
     end
   end
 
+  def validate_person_age_matches_economic_status(record, person_num)
+    age = record.public_send("age#{person_num}")
+    economic_status = record.public_send("ecstat#{person_num}")
+    return unless age && economic_status
+
+    if age < 16 && !tenant_is_economic_child?(economic_status)
+      record.errors.add "ecstat#{person_num}", I18n.t("validations.household.ecstat.child_under_16", person_num:)
+      record.errors.add "age#{person_num}", I18n.t("validations.household.age.child_under_16", person_num:)
```

Comment:
I saw that there are variables in the strings in the yml
I would have expected here that it would read `person_num: person_num` with the first symbol being the variable name for the yml string and the second variable being the value 1,2,3,etc
I guess there is some ruby/rails magic going on here. Can you shed any light on that for me?

Reply:
Hm, yup, I think we can omit it if the argument name is the same as parameter name: https://rubyreferences.github.io/rubychanges//3.1.html#values-in-hash-literals-and-keyword-arguments-can-be-omitted

Reply:
very nice